addon/components/ember-slider.js

```diff
@@ -24,27 +31,35 @@ export default Ember.Component.extend({
 				self.set('value', ui.value);
 			},
 			change : function(event, ui) {
-				(target.sendAction) ? target.sendAction('changeAction', ui.value) :	target.send(self.get('changeAction'), ui.value);
+				(target.sendAction) ? target.sendAction('changeAction', ui.value) : target.send(self.get('changeAction'), ui.value);
 			}
 		});
 
 		this.registerListeners();
 	},
+	
+	proxySlider: function (target, key) {
+		this.$().slider('option', key, this.get(key));
+	},
 
 	registerListeners: function () {
-		var props = ['animate', 'disabled', 'max',
-			'min', 'orientation', 'range', 'step',
-			'value', 'values'];
 		/*jshint loopfunc:false*/
-		for (var i = 0; i < props.length; i++) {
-			this.addObserver(props[i], this, function (target, key) {
-				this.$().slider('option', key, this.get(key));
-			}.bind(this));
+		for (var i = 0, len = props.length; i < len; i++) {
+			this.addObserver(props[i], this, this.proxySlider);
```

Comment:
I need to test this but I think that `this.proxySlider` will not have the correct `this`